EventListener/SetThemeEventListener.php

```diff
@@ -38,7 +38,7 @@ public function __construct(ActiveTheme $activeTheme)
      *
      * @param GetResponseEvent $event
      */
-    public function setActiveThemeOnRequest(GetResponseEvent $event)
+    public function setActiveThemeOnRequest(GetResponseEvent $event): void
```

Comment:
sorry this is a bc break we can not do this

Reply:
😢

Reply:
i decided that im willing to make this break 😜

Reply:
👿